docs/pages/web/pulsar.js

```diff
@@ -2,17 +2,26 @@
 import { type Node } from 'react';
 import AccessibilitySection from '../../docs-components/AccessibilitySection.js';
 import docgen, { type DocGen } from '../../docs-components/docgen.js';
```

Comment:
Should we recommend Pulsar sizes for size components? 

IconButton sm >> 50
Button lg >> 60

Otherwise the size guidance is a little wably

Reply:
Good question. Probably worth bringing up in the team channel?